vignettes/approx-inference.Rmd

```diff
@@ -99,9 +99,16 @@ library(purrr)
 library(tidyr)
 library(tibble)
 library(tidybayes)
+library(cmdstanr)
 ```
 
-First, we begin by simulating data.
+To access the approximate inference methods used in this vignette we will need to use the `cmdstanr` backend for `brms` (we generally recommend using this backend for fitting models). To do this, we first need to install CmdStan (see the README for more details). We can check we have everything we need as follows:
```

Comment:
Suggest newline for new sentence. Easier to edit / see changes

Reply:
I had a look at the package and it looks like this is quite inconsistent so perhaps we need a new issue to address.